app/build.gradle.kts

```diff
@@ -215,26 +214,6 @@ knit {
     }
 }
 
-val ciBuildProperty = "ci-build"
```

Comment:
The parameter `ci-build` is also used in `nightlyReports.yml`, I think it can be removed from this file too.